### PR TITLE
[PREP-278] Fix inconsistensies on copyrightHolders

### PR DESCRIPTION
**Pull request by hmoco**
## Purpose
Fix licenseChanged behavior to be more accurate.

## Changes
Fix inconsistencies on copyrightHolders

### CODE REVIEW

File: app/controllers/submit.js [hunk 1]
```diff
@@ -600,7 +600,7 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
                 this.set('basicsLicense', {
                     licenseType: license || this.get('availableLicenses').toArray()[0],
                     year: this.get('model.licenseRecord') ? this.get('model.licenseRecord').year : false || date.getUTCFullYear().toString(),
-                    copyrightHolders: this.get('model.licenseRecord') ? this.get('model.licenseRecord').copyright_holders.join(', ') : false || ''
+                    copyrightHolders: this.get('model.licenseRecord') ? this.get('model.licenseRecord').copyright_holders.join(',') : false || ''
```
Comment (pattisdr) [inline app/controllers/submit.js]: Can you explain why you need `false || ''` ? Why not just `''`?

**Comment by pattisdr** [comment]
@hmoco - Looks good, just one question.

File: app/controllers/submit.js (hunk 1 above)
Comment (hmoco) [inline app/controllers/submit.js]: hah, good catch! Should be just `''`

**Comment by pattisdr** [comment]
Looks good to me!  The combination of this and the ember-osf PR seems to do the trick. The save button / discard changes are behaving how I would expect. Good work.

**Pull request merged**